numpy/core/umath.py

```diff
@@ -8,9 +8,7 @@
 
 from . import _multiarray_umath
 from ._multiarray_umath import *  # noqa: F403
-from ._multiarray_umath import (
-    _UFUNC_API, _add_newdoc_ufunc, _ones_like
```

Comment:
This is the offending change.

I think we need clear comments in this file and the multiarray one to say that the underscore imports are to some extent considered public.

Reply:
fixing